Sources/AppStoreConnectCLI/Commands/BundleIds/ModifyBundleIdCommand.swift

```diff
@@ -22,12 +22,14 @@ struct ModifyBundleIdCommand: CommonParsableCommand {
     var name: String
 
     func run() throws {
-        let api = try makeService()
+        let service = try makeService()
 
-        _ = try api
+        let result = try service
             .bundleIdResourceId(matching: identifier)
```

Comment:
Do we want to make the most of `awaitResult` and do something like this?
```suggestion
        let bundleIdResult = try service.bundleIdResourceId(matching: identifier).awaitResult()
        let modifyResult = service.request(APIEndpoint.modifyBundleId(id: bundleIdResult, name: self.name))
            .map(BundleId.init)
            .awaitResult()
```

Reply:
@DechengMa I don't think so, keep in mind `awaitResult` actually gives a swift `Result` type
If we wanted to do something like this maybe we could make it an `await` function that throws...
🤔

Reply:
Going give this a go and compare...